apps/demos/utils/visual-tests/matrix-test-helper.js

```diff
@@ -54,6 +69,33 @@ export function getChangedFiles() {
     && JSON.parse(readFileSync(changedFilesPath));
 }
 
+export function globalReadFrom(basePath, relativePath, mapCallback) {
+  const absolute = join(basePath, relativePath);
+  if (existsSync(absolute)) {
+    const result = readFileSync(absolute, 'utf8');
+    return (mapCallback && result && mapCallback(result)) || result;
+  }
+  return null;
+}
+
+export function changeTheme(dirName, relativePath, demoPath, theme) {
+  if (!theme || theme === THEME.generic) {
+    return;
+  }
+
+  const indexFilePath = join(dirName, `${relativePath}${demoPath}/index.html`);
```

Comment:
When you use `path.join` - no need to build strings. All strings anyway will be splitted by slashes/backslashes and rebuilded using `path.sep`. 
```suggestion
  const indexFilePath = join(dirName, relativePath, demoPath, 'index.html');
```

Reply:
https://github.com/DevExpress/DevExtreme/pull/26748/commits/5099ddceea7e349f16bf267aaa3e4127e5ef32bf